Packs/Campaign/ReleaseNotes/3_4_6.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### FindEmailCampaign
+
+- The script will now no longer run in a separate docker container.
```

Comment:
this isnt accurate. you mean maybe the docker container wont get killed after each run?